content/zh/docs/contribute/style/hugo-shortcodes/index.md

```diff
@@ -224,7 +224,7 @@ This is **some markdown.**
 {{< tabs name="tab_with_md" >}}
 {{% tab name="Markdown" %}}
 This is **some markdown.**
-{{< note >}}**Note:** It can even contain shortcodes.{{< /note >}}
+{{< note >}}It can even contain shortcodes.{{< /note >}}
```

Comment:
can you please translate the content as well?

Reply:
done